### PR TITLE
fix(oie): added persona logo for SIW V3

### PR DESCRIPTION
**Pull request by praveendurai-okta**
## Description:

Add persona logo for SIW V3.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [x] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-801591](https://oktainc.atlassian.net/browse/OKTA-801591)

### Screenshot/Video:

Screenshot 2024-09-11 at 10 12 20 AM

### Downstream Monolith Build:

https://bacon-go.aue1e.saasure.net/commits?artifact=monolith&page=1&pageSize=6&sha=4bb9bac02690a2dcb2f6b22b8a2e980c252abf69&tab=main

### CODE REVIEW

File: test/testcafe/spec/AuthenticatorIdvView_spec.js [hunk 1]
```diff
@@ -39,10 +39,7 @@ test
     console.log(pageObject.getFormTitle());
     await t.expect(pageObject.getFormTitle()).eql('Verify your identity with Persona');
     await t.expect(pageObject.getPageSubtitle()).eql('This strengthens security by validating your identity with official documentation.');
-    // GEN 3 does not have logo in this redirect page
-    if(!userVariables.gen3) {
-      await t.expect(pageObject.getBeaconSelector()).contains('mfa-idv-persona');
-    }
+    await t.expect(pageObject.getBeaconSelector()).contains('mfa-idv-persona');
```
Comment (shawyu-okta) [inline test/testcafe/spec/AuthenticatorIdvView_spec.js:42]: 💯

File: src/v3/src/util/idxUtils.ts [hunk 2]
```diff
@@ -148,6 +148,10 @@ export const buildAuthCoinProps = (
     return { authenticatorKey: IDX_STEP.PIV_IDP };
   }
 
+  if (nextStep?.name === IDX_STEP.REDIRECT_IDVERIFY) {
+    return { authenticatorKey: nextStep?.idp?.id || '' };
```
Comment (shawyu-okta) [inline src/v3/src/util/idxUtils.ts:152]: It's ok to return `undefined` from this function if no auth coin should appear

File: src/v3/src/util/idxUtils.test.ts [hunk 3]
```diff
@@ -159,6 +159,24 @@ describe('IdxUtils Tests', () => {
...
+          'http://localhost:3000/idp/identity-verification?stateTokenExternalId=bzJOSnhodWVNZjZuVEsrUj',
+        method: 'GET',
+        idp: {
+          id: 'IDV_PERSONA',
+          name: 'Persona',
+        },
+      },
+    };
+    expect(buildAuthCoinProps(transaction)?.authenticatorKey).toBe('IDV_PERSONA');
+  });
```
Comment (shawyu-okta) [inline src/v3/src/util/idxUtils.test.ts:178]: Can you add a test that covers if `nextStep.idp.id` is not set?

File: src/v3/src/util/idxUtils.ts (hunk 2 above)
Comment (praveendurai-okta) [reply src/v3/src/util/idxUtils.ts]: Assigning `authenticatorKey` only when idp.id exist. else the existing flow will return undefined.

**Review by shawyu-okta — APPROVED** [review]
Merge to `7.23` and use a conventional commit message

**Comment by praveendurai-okta** [comment]
Closing this PR as becon is throwing some error related to branch name. Created [new PR](https://github.com/okta/okta-signin-widget/pull/3715) with same commits.

**Pull request closed without merging**